forage-android/src/main/java/com/joinforage/forage/android/network/model/ForageApiResponse.kt

```diff
@@ -1,7 +1,74 @@
 package com.joinforage.forage.android.network.model
 
+import org.json.JSONObject
+
 sealed class ForageApiResponse<out T> {
     data class Success<out T>(val data: T) : ForageApiResponse<T>()
 
-    data class Failure(val message: String) : ForageApiResponse<Nothing>()
+    data class Failure(val httpStatusCode: Int, val errors: List<ForageError>) : ForageApiResponse<Nothing>()
```

Comment:
Any reason to have `httpStatusCode` here if it is included on the `ForageError` objects?

Reply:
Updated form of error here: [522c7d8](https://github.com/teamforage/forage-android-sdk/pull/23/commits/522c7d8102efadda9db0bc19cd411cb5abca26bd) to match previously discussed.